plugins/BEdita/Core/src/Model/Entity/Folder.php

```diff
@@ -110,6 +114,43 @@ protected function _setParentId($parentId)
         return $parentId;
     }
 
+    /**
+     * Getter for `parent_uname` virtual property
+     *
+     * @return string|null
+     */
+    protected function _getParentUname(): ?string
```

Comment:
We should mark the property as virtual in constructor I think.

Reply:
We didn't do it for `parent_id` virtual property and we generally don't need it in folder `meta` response where we already have `path`

A `uname` version of the path could indeed be useful, probably...

Reply:
done